Parsers/ASimAuthentication/Tests/Microsoft_Github_Authentication_SchemaTest.csv

```diff
@@ -0,0 +1,117 @@
+﻿Result,,,,,,Notes
```

Comment:
Authentication test not needed. remove this file

Reply:
Done